### ISSUE TITLE
Installation problem 

### ISSUE THREAD

**Issue by ElvinGuluzade** [sraoss/pg_ivm#65]
Hi, I am working in a Virtual machine on the adress xx.xx.xx.25 and I have installed this extension going over the steps provided in the readme file. However in the last stage when I run CREATE EXTENSION pg_ivm; I get an error indicating the pg_ivm.control does not exist. But when I check the path mentioned in the error, there are a few files like pg_ivm--1.0--1.1.sql  pg_ivm--1.0.sql  pg_ivm--1.1--1.2.sql  pg_ivm--1.2--1.3.sql  pg_ivm--1.3--1.4.sql  pg_ivm--1.4--1.5.sql  pg_ivm.control. Why would the pg_ivm.control not be available? My first instinct is that, the database I am trying to install and implement the extension to is in another server and when it tries to locate the control file, it cannot access the file in the virtual machine.

**Comment by yugo-n** [comment]
>My first instinct is that, the database I am trying to install and implement the extension to is in another server and when it tries to locate the control file, it cannot access the file in the virtual machine.

I guess your instinct is right. If the machine you installed pg_ivm is different than the machine on which your database is working, the error will be raised. You have to install pg_ivm on the machine where PostgreSQL is installed after setting PATH to pg_config of the appropriate PostgreSQL .

**Comment by ElvinGuluzade** [comment]
Thank you for the answer. Do you mean adding the path of pg_config to the PATH in the environment variables? Or there is another way to set the PATH in the PostgreSQL itself?

**Comment by yugo-n** [comment]
> Do you mean adding the path of pg_config to the PATH in the environment variables? 
Yes, I mean adding the path of the directory where pg_config is located to the PATH environment variable.

**Comment by ElvinGuluzade** [comment]
Thank you very much.
